spec/views/login_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+RSpec.describe 'Login page', type: :system do
```

Comment:
Specs file for the integration tests type is `feature`, for example:

```ruby
 describe 'Login page', type: :feature do
  ...
 end
```
Kindly fix the integration tests type and check for other occurrences 👍